maintainers/maintainer-list.nix

```diff
@@ -3725,6 +3725,12 @@
     githubId = 39825;
     name = "Dominik Honnef";
   };
+  doriath = {
```

Comment:
Please put these changes into a separate commit with the title like "maintainers: add doriath"

Reply:
Moved to separate commit.